docs/getting-started/overview.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 1
+id: overview
+---
+
+# Getting Started with Open Feature
+
+## FlagD
+
+TODO
+
+## Open Feature Operator
+
+TODO
```

Comment:
White flagd and OFO are important, I think the top section should be for using the OpenFeature SDK in-process to evaluate flags, that's are primary use case.

Reply:
I guess we would want to have three different getting started guides then?

Reply:
I think so... Often, OFO and Flagd will be used together, but not always. We will clarify in our meeting.